### PR TITLE
docs: Add 'Code submission'

### PR DESCRIPTION
**Pull request by mrshu**
* Add a quick document on how to format the code for submission.

* Fix some small formatting issues in `how_to.md`.

Signed-off-by: mr.Shu <mr@shu.io>

### CODE REVIEW

**Review by Adman — APPROVED** [review]

File: docs/docs/code_submission.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+├── robot2/
+│   └── robot2.py
+└── robot3/
+    └── robot3.py
+```
+
+To submit it for the competition, there are two things we need to do:
+1. Add a team name
+2. Create a ZIP archive
+
```
Comment (dbscoach) [inline docs/docs/code_submission.md:26]: 3. Optionally add a Team Logo that will be shown on the cube of your robots.

File: docs/docs/code_submission.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+controllers/
+├── robot1/
+│   └── robot1.py
+│   └── utils.py
+├── robot2/
+│   └── robot2.py
+├── robot3/
+│   └── robot3.py
+└── team_name.txt
+```
```
Comment (dbscoach) [inline docs/docs/code_submission.md:48]: └── logo.png

**Review by dbscoach — COMMENTED** [review]
That looks good to my eyes. I'd like to also offer the option to upload a logo or should we avoid that?

Cheers,
David

**Comment by mrshu** [comment]
Thanks @dbscoach , that made sense so I did add it in!

**Pull request merged**